### ISSUE TITLE
cmd button crashes Processing3 pde

### ISSUE THREAD

**Issue by w4nderlust** [sojamo/controlp5#49]
If I press the **cmd** button (Mac OS X 10.11) on my Processing3 application using controlP5 it creashes with the following error:

```
java.lang.ArrayIndexOutOfBoundsException: 768
	at controlP5.ControlWindow.keyEvent(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at processing.core.PApplet$RegisteredMethods.handle(PApplet.java:1395)
	at processing.core.PApplet.handleMethods(PApplet.java:1590)
	at processing.core.PApplet.handleKeyEvent(PApplet.java:2945)
	at processing.core.PApplet.dequeueEvents(PApplet.java:2602)
	at processing.core.PApplet.handleDraw(PApplet.java:2410)
	at processing.javafx.PSurfaceFX$1.handle(PSurfaceFX.java:75)
	at processing.javafx.PSurfaceFX$1.handle(PSurfaceFX.java:1)
	at com.sun.scenario.animation.shared.TimelineClipCore.visitKeyFrame(TimelineClipCore.java:226)
	at com.sun.scenario.animation.shared.TimelineClipCore.playTo(TimelineClipCore.java:184)
	at javafx.animation.Timeline.impl_playTo(Timeline.java:176)
	at javafx.animation.AnimationAccessorImpl.playTo(AnimationAccessorImpl.java:39)
	at com.sun.scenario.animation.shared.InfiniteClipEnvelope.timePulse(InfiniteClipEnvelope.java:110)
	at javafx.animation.Animation.impl_timePulse(Animation.java:1102)
	at javafx.animation.Animation$1.lambda$timePulse$26(Animation.java:186)
	at javafx.animation.Animation$1$$Lambda$101/60587559.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at javafx.animation.Animation$1.timePulse(Animation.java:185)
	at com.sun.scenario.animation.AbstractMasterTimer.timePulseImpl(AbstractMasterTimer.java:344)
	at com.sun.scenario.animation.AbstractMasterTimer$MainLoop.run(AbstractMasterTimer.java:267)
	at com.sun.javafx.tk.quantum.QuantumToolkit.pulse(QuantumToolkit.java:521)
	at com.sun.javafx.tk.quantum.QuantumToolkit.pulse(QuantumToolkit.java:505)
	at com.sun.javafx.tk.quantum.QuantumToolkit.lambda$runToolkit$400(QuantumToolkit.java:334)
	at com.sun.javafx.tk.quantum.QuantumToolkit$$Lambda$56/1178450398.run(Unknown Source)
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
handleDraw() called before finishing
Could not run the sketch (Target VM failed to initialize).
For more information, read revisions.txt and Help → Troubleshooting.
```

**Comment by sojamo** [comment]
This issue is related to the new FX2D renderer, the same effect can't be replicated with other renderers. the command key code in FX2D is 768 for all the other renderers it is 157, the way keys are handled inside controlP5 only allow key codes up to 525, I keep that in mind and will fix it with the next release.

**Comment by sojamo** [comment]
issue fixed with version 2.2.6.
